jest.config.v2.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       testURL: "http://localhost",
       transform: {
         "\\.(gql|graphql)$": "jest-transform-graphql",
-        ".(ts|tsx|js|jsx)": "babel-jest",
+        "^.+\\.(ts|tsx|js|jsx)$": "babel-jest",
```

Comment:
this regex feels weird.. the `$` at the end makes sense, and should be enough to exclude the json files. the beginning part seems unnecessary?

Reply:
I don't know, but I'm trying to apply this here: [b5a4584](https://github.com/artsy/force/pull/9797/commits/b5a458433ba7f97eb767004e8969028bb471031c) 🤞